improv/nexus.py

```diff
@@ -600,24 +615,47 @@ def _startStoreInterface(self, size):
         """
         if size is None:
             raise RuntimeError("Server size needs to be specified")
-        try:
-            self.store_loc = str(os.path.join("/tmp/", str(uuid.uuid4())))
-            self.p_StoreInterface = subprocess.Popen(
-                [
-                    "plasma_store",
-                    "-s",
-                    self.store_loc,
-                    "-m",
-                    str(size),
-                    "-e",
-                    "hashtable://test",
-                ],
-                stdout=subprocess.DEVNULL,
-                stderr=subprocess.DEVNULL,
-            )
-            logger.info("StoreInterface start successful: {}".format(self.store_loc))
-        except Exception as e:
-            logger.exception("StoreInterface cannot be started: {}".format(e))
+        self.use_redis = False
```

Comment:
Right - we assume we're not using Redis, and then if it's found in the config file we flip the flag. This is mostly so that things after the call to `_startStoreInterface` know that they should be using Redis (or, more importantly, they should _not_ be using Plasma). I'd expect this to go away once we deprecate support for Plasma and Nexus doesn't have to know which it's using.

Reply:
Since Redis is the default, I went ahead and removed this - nothing should be checking this flag anymore, and usage of Redis should be assumed unless the Plasma flag is enabled.